cmd/restic/integration_test.go

```diff
@@ -1086,6 +1126,61 @@ func TestPrune(t *testing.T) {
 	testRunCheck(t, env.gopts)
 }
 
+type damagedRepository struct {
+	restic.Repository
+}
```

Comment:
Similarly, I suppose this can be replaced by a damagedBackend.

Reply:
This test now just deletes the file from the backend similar to the test for backup self-healing. That way wrapping the repository or the backend is no longer necessary.